src/test/java/org/folio/rest/api/RequestsApiTest.java

```diff
@@ -97,6 +100,13 @@ public void canCreateARequest()
     DateTime requestExpirationDate = new DateTime(2017, 7, 30, 0, 0, DateTimeZone.UTC);
     DateTime holdShelfExpirationDate = new DateTime(2017, 8, 31, 0, 0, DateTimeZone.UTC);
 
+    UUID isbnIdentifierId = UUID.randomUUID();
+    UUID bnbIdentifierId = UUID.randomUUID();
```

Comment:
What is a BNB identifier?

Reply:
I've found it on a sample instance, I'm not sure what this means 😄.

Reply:
I suggest using a more common identifier type for this example, like ISSN

Reply:
Looks like it is the British National Bibliography ID: https://www.bl.uk/collection-metadata/metadata-services

I learned something new. Good thing I still get these PR emails. 😄

Reply:
@mreno-EBSCO  It is probably time to review the circulation code owners. If you or @mtraneis want to leave the group, please let me know (I'm happy for you both to remain)